td.vue/src/desktop/menu.js

```diff
@@ -17,6 +17,7 @@ import deu from '@/i18n/de.js';
 import ell from '@/i18n/el.js';
 import eng from '@/i18n/en.js';
 import id from '@/i18n/id.js';
```

Comment:
ideally both id.js and ms.js would be imported in alphabetical order, so could these both be moved to after hi.js